driver/config/config.go

```diff
@@ -82,6 +82,7 @@ const (
 	ViperKeyHasherArgon2ConfigParallelism                           = "hashers.argon2.parallelism"
 	ViperKeyHasherArgon2ConfigSaltLength                            = "hashers.argon2.salt_length"
 	ViperKeyHasherArgon2ConfigKeyLength                             = "hashers.argon2.key_length"
+	ViperKeyPasswordAPIDNSName                                      = "selfservice.methods.password.config.api_dns_name"
```

Comment:
Can we rename this to:

```suggestion
	ViperKeyPasswordAPIDNSName                                      = "selfservice.methods.password.config. haveibeenpwned_url"
```